README.md

```diff
@@ -159,6 +159,13 @@ The following changes are now available in production:
 
 ### property-business-api
 
+The following changes are now available in production:
+
+* Minor clarifications made to developer hub documentation:
+  * Updated description of Amend a Historic FHL UK Property Income & Expenses Period Summary endpoint.
```

Comment:
To avoid the repetition of "Updated description of", could this be changed to sm like:
```
* Minor clarifications made to the following endpoint descriptions in the developer hub documentation:
  * Amend a Historic FHL UK Property Income & Expenses Period Summary
```

Reply:
(Also just as it's been done elsewhere, the endpoint name should probably be in tick-marks)